pattern-library/html/boilerplate-blank-RTL.html

```diff
@@ -0,0 +1,54 @@
+<!--
+edX Pattern Library: Boilerplate HTML Template: Header + Footer (RTL layout direction!)
+
+NOTES:
+
+* base template with details on how to handle primary/secondary assets, metadata, and basic view-level UI (on canvas vs. conditional and off-canvas elements)
+* right-to-left (RTL) layout configuration have been added
+* see https://github.com/edx/ux-pattern-library/wiki/Styleguide:-HTML#right-to-left-rtl-support for more information
+-->
```

Comment:
Great point. I've moved all the comments beneath the DOCTYPE to keep them valid. I considered adding a comment to remove it, but that felt redundant as most of the comments in the file should be removed.